lambda/README.md

```diff
@@ -33,20 +32,32 @@ AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws es \
 --ebs-options EBSEnabled=true,VolumeType=standard,VolumeSize=10
 ```
 
-1. Call the function
+1. Create the SNS Topic
 ```
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws lambda \
---endpoint-url=http://localhost:4574 invoke \
---function-name f1 result.log \
---payload '{"Records": [{"EventSource": "foo", "Sns": { "Timestamp": "2014-05-16T08:28:06.801064-04:00", "Message": "Hello world!" }}]}'
+awslocal sns create-topic --name rialto-msg
+```
+
+1. Subscribe the lamba function to the SNS Topic
+```
+awslocal sns subscribe \
+  --topic-arn arn:aws:sns:us-east-1:123456789012:rialto-msg \
+  --protocol lambda \
+  --notification-endpoint arn:aws:lambda:localstack:000000000000:function:f1
+```
+
+1. Publish to the SNS Topic
+```
+awslocal sns publish \
+   --topic-arn arn:aws:sns:us-east-1:123456789012:test-topic \
```

Comment:
Why are you sending a message to a topic you haven't subscribed to?

Reply:
copy-pasta, fixing.